### ISSUE TITLE
FoolCraft DevTeam Req: Block Armor Config File Not Behaving

### ISSUE THREAD

**Issue by Jo-Mamma75** [2piradians/Block-Armor#6]
So another request from the FoolCraft DevTeam.

We wanted to be able to disable certain armor sets on a one-by-one basis via the config file but, if we do so and load the game, all changes are reverted back and all armor is enabled again.  I personally did not play with the config file (CriticalMole did) so I suppose it's possible that he may have missed something in the 3000+ lines of the config file.  I just wanted to ask if this was an issue that you've seen before and if it can be fixed.

Long story short, due to the number of items in the game, there is an absolute crap-ton of armor and we wanted to trim the list down a little bit without disabling the armor sets from any one mod completely.

Thanks again for being so helpful.  We all sincerely appreciate it.

Jo_Mamma75
Iskall85 Community Administrator
FoolCraft DevTeam

**Comment by Furgl** [comment]
This should be fixed in v2.4.3
